nms/app/packages/magmalte/alerts/lteAlerts.js

```diff
@@ -66,5 +82,20 @@ export default function getLteAlerts(
       labels: {severity: 'minor'},
       annotations: {description: 'Alerts when we have UE attach failures'},
     },
+    'Gateway Checkin Failure': {
+      alert: 'Gateway Checkin Failure',
+      expr: `checkin_status{networkID=~"${networkID}" < 1`,
+      labels: {severity: 'critical'},
+      annotations: {description: 'Alerts when we have gateway checkin failure'},
+    },
+    'Dip in Connected UEs': {
+      alert: 'Dip in Connected UEs',
+      expr: `(ue_connected{networkID=~"${networkID}"} - ue_connected{networkID=~"${networkID}"} offset 5m) / (ue_connected{networkID=~"${networkID}"}) < -0.5`,
+      labels: {severity: 'critical'},
+      annotations: {
+        description:
+          'Alerts when there is a sudden dip in number of connected UEs',
```

Comment:
Could this be updated to be specific about it being a 50% dip in UEs over 5 minutes?